### ISSUE TITLE
Evaluation details about Qwen2-VL-72B

### ISSUE THREAD

**Issue by lan-lw** [THUDM/LVBench#7]
Thank you for your interesting work!

For Qwen2-VL-72B model, are you using the whole video or just sample 48 frames per video? If you used the whole video, what's the fps?

**Comment by huangshiyu13** [comment]
We use 48 frames for each video.

**Comment by lan-lw** [comment]
Thank you for your response.

Are you uniformly sampling 48 frames?  I am wondering for tasks like temporal grounding, are 48 frames enough for precise localization.

**Comment by huangshiyu13** [comment]
> Thank you for your response.
> 
> Are you uniformly sampling 48 frames? I am wondering for tasks like temporal grounding, are 48 frames enough for precise localization.

Yes, we uniformly sample 48 frames. We used 48 frames because we found that the maximum number of frames we can input is only 48 frames, and exceeding 48 frames can easily cause memory explosion.

**Comment by IssacCyj** [comment]
> > Thank you for your response.
> > Are you uniformly sampling 48 frames? I am wondering for tasks like temporal grounding, are 48 frames enough for precise localization.
> 
> Yes, we uniformly sample 48 frames. We used 48 frames because we found that the maximum number of frames we can input is only 48 frames, and exceeding 48 frames can easily cause memory explosion.

Just double check, are you sampling 48 frames from the hour-long video or I misunderstood the answer. This mean you only use 1 frame at every 1-2 minutes video clip which sounds confusing to me, given that the temporal grounding task has second-level annotations.

**Comment by huangshiyu13** [comment]
> > > Thank you for your response.
> > > Are you uniformly sampling 48 frames? I am wondering for tasks like temporal grounding, are 48 frames enough for precise localization.
> > 
> > 
> > Yes, we uniformly sample 48 frames. We used 48 frames because we found that the maximum number of frames we can input is only 48 frames, and exceeding 48 frames can easily cause memory explosion.
> 
> Just double check, are you sampling 48 frames from the hour-long video or I misunderstood the answer. This mean you only use 1 frame at every 1-2 minutes video clip which sounds confusing to me, given that the temporal grounding task has second-level annotations.

Yes, the Qwen2-VL-72B model samples an average of 48 frames per video. The main reason for this is that we cannot feed more frames into the deployed model. You can submit the evaluation results to us if you can evaluate Qwen2-VL-72B using more frames. After reviewing, we can update the leaderboard with the new results.

**Comment by IssacCyj** [comment]
thanks for the quick response!

**Comment by IceFlameWorm** [comment]
> > > > Thank you for your response.
> > > > Are you uniformly sampling 48 frames? I am wondering for tasks like temporal grounding, are 48 frames enough for precise localization.
> > > 
> > > 
> > > Yes, we uniformly sample 48 frames. We used 48 frames because we found that the maximum number of frames we can input is only 48 frames, and exceeding 48 frames can easily cause memory explosion.
> > 
> > 
> > Just double check, are you sampling 48 frames from the hour-long video or I misunderstood the answer. This mean you only use 1 frame at every 1-2 minutes video clip which sounds confusing to me, given that the temporal grounding task has second-level annotations.
> 
> Yes, the Qwen2-VL-72B model samples an average of 48 frames per video. The main reason for this is that we cannot feed more frames into the deployed model. You can submit the evaluation results to us if you can evaluate Qwen2-VL-72B using more frames. After reviewing, we can update the leaderboard with the new results.

So it seems quite doultful for the Qwen2-VL-72B results on these second-level tasks. If most part of the movie content is dropped after downsampling, I guess there is no better explanations than “hallucination” for the model outputs same as the ground truth
